actions/lib/icinga2.py

```diff
@@ -0,0 +1,41 @@
+import requests
+from icinga2api.client import Client
+from st2common.runners.base_action import Action
+requests.packages.urllib3.disable_warnings()   # pylint: disable=no-member
+
+try:
+    import urllib3
+    urllib3.disable_warnings()
+except ImportError:
+    pass
+
+
+class Icinga2Action(Action):
+
+    def __init__(self, config):
+        super(Icinga2Action, self).__init__(config)
+        self.myconfig = config
+        self._client = self._get_client()
+
+    def _get_client(self):
+        api_url = self.myconfig['api_url']
+
+        if ('api_user' in self.myconfig and
```

Comment:
Could probably simplify that and do ``api_user = self.myconfig.get('api_user', None)`` and then ``if api_user ...``.

Same for password check.

Reply:
Reworked that in 
https://github.com/StackStorm-Exchange/stackstorm-icinga2/pull/9/commits/a8024ed24ed0e8b273af63f892eecb63041c32b7

What do you think?